app/scripts/lib/constants.js

```diff
@@ -29,6 +29,8 @@ define(function (require, exports, module) {
 
     // Users that sign in to the content server directly
     CONTENT_SERVER_CONTEXT: 'web',
+
+    FX_SYNC_CONTEXT: 'fx_sync',
```

Comment:
It's a generic context name, for fx-sync broker, which is the basis of others. Shouldn't be used by external reliers.